client/src/Components/Mentors.js

```diff
@@ -72,12 +91,17 @@ export default function Mentors(props) {
 	if (props.questions) {
 		return (
 			<div className="row">
+				<div className='filterButtons col-6'>
+					{tagsCollection.map((tag)=>{
+						return(<button>{tag}</button>);
+					})}
+				</div>
```

Comment:
can you please add key to fix the warning

Reply:
I have new code waiting to be pushed will fix in next iteration. Unfortunately there are more changes i dont want to commit just now